src/main/java/frc/robot/subsystems/chassis/Chassis.java

```diff
@@ -397,5 +458,14 @@ public void initSendable(SendableBuilder builder) {
                 module.calibrateOffset();
             }
         }).ignoringDisable(true));
+
+        Utils.addDoubleProperty(builder, "Velocity", () -> {
+            return getVelocity().getNorm();
+        }, 2);
+        Utils.addDoubleProperty(builder, "Velocity Angle", () -> {
+            return getVelocity().getAngle().getDegrees();
+        }, 2);
+
+        builder.addBooleanProperty("Is Break", () -> isBreak, null);
```

Comment:
utils

Reply:
If you mean that all the other properties are added through Utils, this is because it adds them rounded to 2 places after decimal, a boolean doesn't have that issue